channels/ca.m3u

```diff
@@ -1,5 +1,35 @@
 #EXTM3U x-tvg-url="http://epg.streamstv.me/epg/guide-canada.xml.gz"
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/Jxjg4Nx.png" group-title="",5AAB Music Channel
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/x4r2oCs.jpg" group-title="",CBC Calgary
+http://ummodigital.com:8000/jorge1/jorge1/5863
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/bbmuFF4.jpg" group-title="",CBC Charlottetown
+http://ummodigital.com:8000/jorge1/jorge1/5864
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/qhgBBkR.png" group-title="",CBC Edmonton
+http://ummodigital.com:8000/jorge1/jorge1/5865
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/bbmuFF4.jpg" group-title="",CBC Fredericton
+http://ummodigital.com:8000/jorge1/jorge1/5866
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/bbmuFF4.jpg" group-title="",CBC Halifax
+http://ummodigital.com:8000/jorge1/jorge1/5867
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/6VDawUA.jpg" group-title="",CBC Montreal
+http://ummodigital.com:8000/jorge1/jorge1/5868
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/FTh7htR.png" group-title="",CBC Ottawa
+http://ummodigital.com:8000/jorge1/jorge1/5742
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/bbmuFF4.jpg" group-title="",CBC Regina
+http://ummodigital.com:8000/jorge1/jorge1/5869
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/bbmuFF4.jpg" group-title="",CBC St John
+http://ummodigital.com:8000/jorge1/jorge1/5870
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/Mmsz8RP.jpg" group-title="",CBC Toronto
+http://ummodigital.com:8000/jorge1/jorge1/5862
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/GUGNts3.jpg" group-title="",CBC Vancouver
+http://ummodigital.com:8000/jorge1/jorge1/5871
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/5n9cgt9.png" group-title="",CBC Windsor
+http://ummodigital.com:8000/jorge1/jorge1/5872
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/bbmuFF4.jpg" group-title="",CBC Winnipeg
+http://ummodigital.com:8000/jorge1/jorge1/5873
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/bbmuFF4.jpg" group-title="",CBC Yellowknife
+http://ummodigital.com:8000/jorge1/jorge1/5874
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/HmqRMIq.png" group-title="Sport",Fubo Cycling
+http://ummodigital.com:8000/jorge1/jorge1/3426
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/k9M5ydW.png" group-title="",5AAB Music Channel
```

Comment:
As I understand it, all links starting with "http://ummodigital.com:8000/jorge1/jorge1" do not work

Reply:
not working. this link from provider and seems was deleted

Reply:
Strange, when i open these links in VLC they all play

Reply:
@boomski well what are we going to do? add to playlist or not?

Reply:
I tried to open them in kodi and then they also don't play , but for me in vlc they play fine.
But if you or anyone else can't play them i would not add them to the list.

Reply:
Ok, then I close this PR for now.